src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const filterMovies = (currentMovies: Movie[]) => {
+    return currentMovies.filter(curentMovie => (
+      // eslint-disable-next-line max-len
+      (curentMovie.title.toLowerCase().includes(searchQuery.toLowerCase().trim()))
+      // eslint-disable-next-line max-len
+      || (curentMovie.description.toLowerCase().includes(searchQuery.toLowerCase().trim()))
+    ));
```

Comment:
```suggestion
		const prettyQuery = searchQuery.toLowerCase().trim();
		
    return currentMovies.filter({ title, description } => (
      (title.toLowerCase().includes(prettyQuery))
      || (description.toLowerCase().includes(prettyQuery))
    ));
```

better refactor the places in code, where you try to use `// disable max-len` 😉